packages/next/src/build/webpack-config.ts

```diff
@@ -915,12 +924,9 @@ export default async function getBaseWebpackConfig(
 
   const reactServerCondition = [
     'react-server',
-    ...mainFieldsPerCompiler[
-      isEdgeServer ? COMPILER_NAMES.edgeServer : COMPILER_NAMES.server
-    ],
-    'node',
-    'import',
-    'require',
+    ...(isEdgeServer ? edgeConditionNames : []),
+    // inherits the default conditions
+    '...',
```

Comment:
What's up with the triple dots? Is this a webpack pattern?

Reply:
Yes, consulted @sokra it means in the parent one, which based on the import syntax (use import or require), so we don't need to hardcode it